docs/rules/no-namespace.md

```diff
@@ -25,6 +31,22 @@ import * as foo from 'foo';
 import defaultExport, * as foo from 'foo';
 ```
 
+```js
+// { allow: ['foo'] }
+import * as bar from 'bar';
+
+// { forbid: ['foo'] }
+import * as foo from 'foo';
+```
+
+### Options
+
+This rule takes the following options:
+
+`allow`: an array of namespaces the rule allows
```

Comment:
i assume this should cover *specifiers*, not the binding names?

(having the binding name and the specifier be the same in most of the examples make it unclear to which this applies)

Reply:
the allow option works as follows: 
```
// { allow: ['myModule', 'React']
import * as React from 'react';
import * as myModule from '../module/randomFile.js';
```